Core/Src/fault.c

```diff
@@ -1,3 +1,45 @@
 #include "fault.h"
 
 osMessageQueueId_t fault_handle_queue;
+
+//Function to determine priority base on severity
+void priority(void) {
+    osThreadId_t id;
+    MSGQUEUE_OBJ_t msg;
```

Comment:
So the type of the message that we receive depends on what we are actually queuing, in this case I believe the type is `FaultStatus_t` or something I forgot exactly what it is